tests/integration/api/controllers/monitoring.spec.js

```diff
@@ -136,7 +136,7 @@ describe('monitoring', () => {
           count: 0,
         },
         connected_users: {
-          count: 1, //not logged in browser
+          count: 0, //not logged in browser
```

Comment:
Do you know what caused this change?  I must be missing something, but none of the changes to the implementation code seem like they would cause this....

Reply:
Ahh, got it! :+1: